server/service/src/purchase_order_line/update/test.rs

```diff
@@ -31,6 +31,10 @@ mod update {
                     id: "purchase_order_line_id_1".to_string(),
                     purchase_order_id: "test_purchase_order_a".to_string(),
                     item_id: mock_item_a().id.to_string(),
+                    requested_pack_size: None,
```

Comment:
This and similar can probably be `..Default::default()` which will fill in all the defaults.
For tests this generally means less updates next time there's a change..